improver/wxcode/wxcode_decision_tree.py

```diff
@@ -192,19 +199,19 @@ def wxcode_decision_tree():
                                        (1.0, 'mm hr-1')]],
             'diagnostic_conditions': [['above', 'above']]},
 
+        # B.b
         'light_precipitation': {
             'succeed': 'light_precipitation_cloud',
             'fail': 'drizzle_mist',
-            'probability_thresholds': [0.5, 0.5],
+            'probability_thresholds': [0.5],
             'threshold_condition': '>=',
-            'condition_combination': 'OR',
+            'condition_combination': '',
```

Comment:
Does this need to be defined as an empty string, or can it just be missing?

Reply:
It needs to be set, unless I modify the weather_symbols code to use dictionaries differently.

Reply:
OK.